client/utils/utils.go

```diff
@@ -15,57 +15,35 @@ import (
 // DefaultGasAdjustment is applied to gas estimates to avoid tx
 // execution failures due to state changes that might
 // occur between the tx simulation and the actual run.
-const DefaultGasAdjustment = 1.2
+const DefaultGasAdjustment = 1.0
```

Comment:
Is it useful or maybe even potentially dangerous to have this defined in two places? Seems its already part of the `CLIContext`?

Reply:
It may not just work for staking transactions, where previous txs in the same block would adjust the gas required.

Reply:
I see. What would you suggest then?

Reply:
Well the gas adjustment is part of the `CLIContext`. Why not just set it upstream (where the default can be used)? Is that not possible?

Reply:
A sane default (i.e. != 0) is required for REST endpoints too.

Reply:
We can really choose any number and revisit during code freeze time.